src/main/java/com/faforever/client/replay/ReplayDetailController.java

```diff
@@ -300,6 +305,15 @@ private void onReplayChanged(ReplayBean newValue) {
     if (newValue.getReplayFile() != null) {
       enrichReplayLater(newValue.getReplayFile(), newValue);
     }
+    
+    newValue.setLeagueScores(null);
+    replayService.getLeagueScoreJournalForReplay(newValue)
+        .thenAccept(scores -> Platform.runLater(() -> {
+          newValue.setLeagueScores(scores);
+          // This looks a bit ugly. Ideally we should wait with drawing the window until we have the league scores,
+          // then we don't need to trigger a redraw here
+          populateTeamsContainer(teams.getValue());
```

Comment:
We shouldn't have to recreate the teamCardControllers again. It should be handled like the showRatingChange where setting a value for the teamCardControllers will trigger the display rather than it needing to be recreated.